### DISCUSSION TITLE
Config/Memories Backup and ALC readings

### DISCUSSION THREAD

**Discussion by PCBenetton** [gdyuldin/x6100_gui#17] [Q&A] [answered]
Hello everybody. 
Yesterday I was playing radio and using the factory FW on the X6100 using only 5W (internal battery) and I was not able to cut through a minor pile-up. Changed to the 0.26.1 FW to test it out and first thing I did was to try to make it through that pile-up (same 5W) and did it pretty fast and received a great report on signal. Not sure if it was luck on changing band conditions (10m at night, it was a ~2500km QSO) or if the FW made a difference. I was very pleased either way.

Anyway, since I was using the 0.26.1 for the first time I was setting it up as I like it two things came to mind:
1 - a config backup feature would be useful for settings and memories, it could extracted before flashing a new FW and later saved to the SD using the DATA partition. (the webserver ideia would work nice with this feature also);
2 - whats the correct ALC reading to go after? I was fiddling with the mic gain trying to max the power output with minimal gain (without and external power meter thats pretty hard) and then I wondered if ALC could help to do it "on the fly", and what is the optimal value or range to strive for when setting up mic gain and digital modes.

Thanks!

**Comment by gdyuldin** [comment, marked as answer]
Hi.
I created a separated thread for Config/Memories Backup - #20 
Regarding ALC - rough estimate of strength of signal suppression for different ALC values is:
- ALC 0 is 0 db
- ALC 3 is -1 db
- ALC 6 is -3 db
- ALC 9 is -10 db

For my opinion - you shouldn't exceed values > 6 for SSB. For digital modes I guess values below 1 is fine.
Also, R1CBU FW have inverted ALC values (0 on R1CBU is the same as 10 on original). ALC values above is from R1CBU FW.
